src/plugins/auto/tests/unit/default_perf_hint_test.cpp

```diff
@@ -125,7 +125,7 @@ class AutoDefaultPerfHintTest : public tests::AutoTest,
         testConfigs.push_back(ConfigParams{
             "AUTO",
             {"GPU"},
-            {{"DEVICE_PROPERTIES", "GPU:{PERFORMANCE_HINT:THROUGHPUT}}"}, {"MULTI_DEVICE_PRIORITIES", "GPU"}}});  // GPU: get perf_hint:tput
+            {{"DEVICE_PROPERTIES", "{GPU:{PERFORMANCE_HINT:THROUGHPUT}}"}, {"MULTI_DEVICE_PRIORITIES", "GPU"}}});  // GPU: get perf_hint:tput
```

Comment:
Please add test data with a secondary property is different from the AUTO properties. Then check if secondary property can overwrite AUTO properties even AUTO properties is updated in AUTO plugin.

Reply:
Update.